ui/src/cells/actions/thunks.ts

```diff
@@ -129,11 +135,37 @@ export const createCellWithView = (
   }
 }
 
+export const createDashboardWithView = async (
+  orgID: string,
+  dashboardName: string,
+  view: View
+): Promise<void> => {
+  try {
+    const newDashboard = {
+      orgID,
+      name: dashboardName,
+      cells: [],
+    }
+
+    const resp = await postDashboard({data: newDashboard})
+
+    if (resp.status !== 201) {
+      throw new Error(resp.data.message)
+    }
+
+    await createCellWithView(resp.data.id, view)
```

Comment:
wait this is supposed to be 'dispatch(createCellWithView())`

Reply:
you're totally right